packages/bsky/src/auto-moderator/abyss.ts

```diff
@@ -69,8 +74,8 @@ export class Abyss implements ImageFlagger {
     return labels
   }
 
-  getReqUrl(params: { did: string }) {
-    return `${this.endpoint}/xrpc/com.atproto.unspecced.scanBlob?did=${params.did}`
+  getReqUrl(params: { did: string; uri: string }) {
+    return `${this.endpoint}/xrpc/com.atproto.unspecced.scanBlob?did=${params.did}&uri=${params.uri}`
```

Comment:
The `did` param should be safe, but I think we'll want to encode the `uri` param with `encodeURIComponent()`.

Reply:
Could also do something along these lines:
```ts
const search = new URLSearchParams(params)
return `${this.endpoint}/xrpc/com.atproto.unspecced.scanBlob?${search.toString()}`
```